rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ds/DataSpaceClient.java

```diff
@@ -89,7 +89,14 @@ public void init(String restServerUrl, ISchedulerClient client) {
                                                                              .useSystemProperties()
                                                                              .build());
         this.restDataspaceUrl = restDataspaceUrl(restServerUrl);
-        this.sessionId = client.getSession();
+        try {
+            this.sessionId = client.getSession();
+        } catch (NotConnectedException e) {
+            if (log.isDebugEnabled()) {
+                log.debug("Error client trying to retrieve session from disconnected client");
```

Comment:
Better message i think is 
"Error : trying to retrieve session from disconnected client."
Also should this beat warn level?